tests/push/test_bulk_push_rule_evaluator.py

```diff
@@ -126,3 +128,77 @@ def test_action_for_event_by_user_disabled_by_config(self) -> None:
         # Ensure no actions are generated!
         self.get_success(bulk_evaluator.action_for_events_by_user([(event, context)]))
         bulk_evaluator._action_for_event_by_user.assert_not_called()
+
+    @override_config({"experimental_features": {"msc3952_intentional_mentions": True}})
+    def test_mentions(self) -> None:
+        """Test the behavior of an event which includes invalid mentions."""
+        bulk_evaluator = BulkPushRuleEvaluator(self.hs)
+
+        sentinel = object()
+
+        def create_and_process(mentions: Any = sentinel) -> bool:
+            content = {}
```

Comment:
Took me a moment to realise what this function does. Maybe a docstring e.g. 

```suggestion
            """Returns true iff the `mentions` trigger an event push action."""
            content = {}
```

would help?